### PR TITLE
osmium: add page

### PR DESCRIPTION
**Pull request by JakobMiksch**
- Add the CLI tool [osmium](https://osmcode.org/osmium-tool/manual)
- Version of the command: `1.14.0`

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/osmium.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# osmium
+
+> Multipurpose tool for handling OpenStreetMap (OSM) files.
+> More information: <https://osmcode.org/osmium-tool/manual>.
+
+- Show file information:
+
+`osmium fileinfo {{path/to/input}}.osm`
```
Comment (kbdharun) [inline pages/common/osmium.md:8]:
```suggestion
`osmium fileinfo {{path/to/input.osm}}`
```

File: pages/common/osmium.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+- Display contents:
+
+`osmium show {{path/to/input}}.osm`
```
Comment (kbdharun) [inline pages/common/osmium.md:12]:
```suggestion
`osmium show {{path/to/input.osm}}`
```

File: pages/common/osmium.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+- Convert file format from PBF into XML:
+
+`osmium cat {{path/to/input}}.osm.pbf -o {{path/to/output}}.osm`
```
Comment (kbdharun) [inline pages/common/osmium.md:16]:
```suggestion
`osmium cat {{path/to/input.osm.pbf}} -o {{path/to/output.osm}}`
```

File: pages/common/osmium.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+- Extract a geographic region by the given bounding box:
+
+`osmium extract -b {{min_longitude}},{{min_latitude}},{{max_longitude}},{{max_latitude}} {{path/to/input}}.pbf -o {{path/to/output}}.pbf`
```
Comment (kbdharun) [inline pages/common/osmium.md:20]:
```suggestion
`osmium extract -b {{min_longitude}},{{min_latitude}},{{max_longitude}},{{max_latitude}} {{path/to/input.pbf}} -o {{path/to/output.pbf}}`
```

File: pages/common/osmium.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+- Display contents:
+
+`osmium show {{path/to/input}}.osm`
+
+- Convert file format from PBF into XML:
+
+`osmium cat {{path/to/input}}.osm.pbf -o {{path/to/output}}.osm`
+
+- Extract a geographic region by the given bounding box:
```
Comment (kbdharun) [inline pages/common/osmium.md:18]:
```suggestion
- Extract a geographic region by the given [b]ounding box:
```

File: pages/common/osmium.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+`osmium extract -b {{min_longitude}},{{min_latitude}},{{max_longitude}},{{max_latitude}} {{path/to/input}}.pbf -o {{path/to/output}}.pbf`
+
+- Extract a geographic region by a GeoJSON file:
+
+`osmium extract -p {{path/to/polygon}}.geojson {{path/to/input}}.pbf -o {{path/to/output}}.pbf`
```
Comment (kbdharun) [inline pages/common/osmium.md:24]:
```suggestion
`osmium extract -p {{path/to/polygon.geojson}} {{path/to/input.pbf}} -o {{path/to/output.pbf}}`
```

File: pages/common/osmium.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+
+- Filter all objects tagged as "restaurant":
+
+`osmium tags-filter {{path/to/input}}.pbf amenity=restaurant -o {{path/to/output}}.pbf`
```
Comment (kbdharun) [inline pages/common/osmium.md:28]:
```suggestion
`osmium tags-filter {{path/to/input.pbf}} amenity=restaurant -o {{path/to/output.pbf}}`
```

File: pages/common/osmium.md [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+
+- Filter for "way" objects tagged as "highway":
+
+`osmium tags-filter {{path/to/input}}.pbf w/highway -o {{path/to/output}}.pbf`
```
Comment (kbdharun) [inline pages/common/osmium.md:32]:
```suggestion
`osmium tags-filter {{path/to/input.pbf}} w/highway -o {{path/to/output.pbf}}`
```

File: pages/common/osmium.md [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+
+- Filter "way" and "relation" objects tagged as "building":
+
+`osmium tags-filter {{path/to/input}}.pbf wr/building -o {{path/to/output}}.pbf`
```
Comment (kbdharun) [inline pages/common/osmium.md:36]:
```suggestion
`osmium tags-filter {{path/to/input.pbf}} wr/building -o {{path/to/output.pbf}}`
```

**Review by kbdharun — CHANGES_REQUESTED** [review]
Thanks for your contribution @JakobMiksch, I have some suggestions for this page. We prefer placing the file extensions inside along with the path for example `{{path/to/file.exe}}`.

Tip: Batch all suggestions from the files tab and make it a single commit. or if you are using GitHub on mobile use the web version of GitHub to do the same.

**Comment by JakobMiksch** [comment]
thanks @kbdharun for your review!
I applied your suggestions :+1:

**Review by kbdharun — APPROVED** [review]

**Review by einverne — APPROVED** [review]
LGTM.

**Pull request merged**